tests/components/holiday/test_config_flow.py

```diff
@@ -0,0 +1,109 @@
+"""Test the Holiday config flow."""
+from unittest.mock import AsyncMock
+
+import pytest
+
+from homeassistant import config_entries
+from homeassistant.components.holiday.const import CONF_PROVINCE, DOMAIN
+from homeassistant.const import CONF_COUNTRY
+from homeassistant.core import HomeAssistant
+from homeassistant.data_entry_flow import FlowResultType
+
+from tests.common import MockConfigEntry
+
+pytestmark = pytest.mark.usefixtures("mock_setup_entry")
+
+
+async def test_form(hass: HomeAssistant, mock_setup_entry: AsyncMock) -> None:
```

Comment:
Can we add an additional test setting HA language to something else than English so we see the title gets translated?

Reply:
Done.